### DISCUSSION TITLE
Alunni che hanno frequentato all'estero

### DISCUSSION THREAD

**Discussion by rodolfo024** [iisgiua/giuaschool#265] [Richieste di aiuto] [answered]
Come bisogna comportarsi per gli alunni che hanno frequentato all'estero o tutto l'anno o solo il secondo periodo? Perché al momento di inserimento dell'utente è richiesto se l'alunno frequenta o meno all'estero. E' implementabile un esito (positivo) "mobilità internazionale"?

**Comment by trinko** [comment, marked as answer]
La gestione della frequenza all'estero è ancora parziale. In sostanza, se il sistema riconosce che l'alunno frequenta all'estero è escluso dallo scrutinio e viene inserito a verbale la seguente frase:

> Il Consiglio di Classe segnala inoltre che l'alunno/a XXX non viene scrutinato/a in quanto frequenta l'anno all'estero, come da documentazione agli atti.

Per il resto, è considerato come un alunno trasferito. La frase si può modificare dai template che iniziano con  **templates/coordinatore/documenti/scrutinio_verbale_**

Per indicare che un alunno frequenta all'estero:

1.  selezionare l'apposita casella presente nell'anagrafica degli alunni
2. impostare l'alunno come trasferito altrove: **ALUNNO > CAMBIO CLASSE > TRASFERISCI ALUNNO**
